src/config/createForm.controller.js

```diff
@@ -31,6 +29,25 @@ export default ({ form, formConfirmation }) => {
   });
 
   form.on("submitDone", () => {
-    if (formConfirmation) window.location.href = formConfirmation;
+    pushDataLayer({
+      event: "formio-submission",
+      submissionsUrl: `form.io: ${form.formio.submissionsUrl}`,
+      "formio-name": form._form.title,
+      "formio-version": form._form.modified,
+    });
+
+    if (formConfirmation)
+      setTimeout(() => {
+        window.location.href = formConfirmation;
+      }, 200);
```

Comment:
I believe this should be ok since its on submitDone, 

i know in the past we ran into issues  (race condition) where we were redirecting without ensuring the Promise callback on submission actually did submit and got a successful response.